specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1221 @@
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Update a Private Endpoint Connection.",
+        "operationId": "PrivateEndpoints_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "PrivateEndpoints_Update": {
+            "$ref": "./examples/PrivateEndpoints_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          },
+          {
+            "name": "PrivateEndpoint",
+            "in": "body",
+            "description": "The Private Endpoint metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "delete": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Deletes a Private Endpoint Connection from the IoT Central Application.",
+        "operationId": "PrivateEndpoints_Delete",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "PrivateEndpoints_Delete": {
+            "$ref": "./examples/PrivateEndpoints_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is returned as a response to the status polling request for the delete operation. The body contains the operation information that tracks the deletion status.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationStatusResult"
+            }
+          },
+          "202": {
```

Comment:
needs x-ms-long-running-operation

Reply:
i placed this property at the api level. is it meant to go on the response?

Reply:
@pilor, i think i see what you mean. most teams put this property at the end of the api definition. i can update to follow that approach.